src/main/java/com/dev/cinema/model/CinemaHall.java

```diff
@@ -0,0 +1,25 @@
+package com.dev.cinema.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    @OneToMany
```

Comment:
not sure we will need this annotation in future, let's remove that for now unless there is a particular reason to keep it

Reply:
ok